.circleci/config.yml

```diff
@@ -136,7 +136,10 @@ commands:
         default: "5.0"
     steps:
       - run: bundle exec pod repo add-cdn jsDelivr-Specs "https://cdn.jsdelivr.net/cocoa/"
```

Comment:
The jsdelivr repo is no longer maintained, we should probably remove that line or change it to `'https://cdn.cocoapods.org/'`